### PR TITLE
Anjaline/better icon

### LINKED ISSUE #458: Better Icon Indicator
image

- If the file is a PDF then use the above icon, otherwise if it is an image, then use a image file icon instead

### PR DESCRIPTION
**Pull request by AnjalineSingh**

### CODE REVIEW

**Comment by AnjalineSingh** [comment]
Screen Shot 2023-09-10 at 9 29 48 PM

File: src/components/Forms/Register.jsx [hunk 1]
```diff
@@ -175,7 +175,7 @@ const Register = () => {
               setUser={setUser}
               text="Upload Resume"
               maxSize={[150, "KB"]}
-              types={["pdf"]}
+              types={["pdf", "jpg", "jpeg", "png"]}
```
Comment (shahdivyank) [inline src/components/Forms/Register.jsx]: resume is pdf only

File: src/components/Forms/Textarea.jsx [hunk 2]
```diff
@@ -37,6 +37,7 @@ const Textarea = ({
         name={name}
         placeholder={placeholder}
         value={value}
+        style={{ resize: "none", height: "50px" }}
```
Comment (shahdivyank) [inline src/components/Forms/Textarea.jsx]: tailwind only

Comment (shahdivyank) [reply src/components/Forms/Textarea.jsx]: is this from the previous PR? make sure to separate it

Comment (AnjalineSingh) [reply src/components/Forms/Textarea.jsx]: yeah sorry it was from the other PR but it ended up here loll but i deleted it and im gonna make sure the resize changes are in the other branch

**Review by menthy-wu — APPROVED** [review]

**Review by shahdivyank — APPROVED** [review]

**Pull request merged**